cmd/cryoutilities/main.go

```diff
@@ -18,25 +18,28 @@ package main
 
 import (
 	"context"
-	"cryoutilities/internal"
 	"errors"
 	"log"
 	"os"
 	"strconv"
 	"strings"
 
+	"github.com/spf13/afero"
+
+	"cryoutilities/internal"
+
 	"github.com/cristalhq/acmd"
 )
 
 func main() {
 	// Delete old log file
-	os.Remove(internal.LogFilePath)
+	func() { _ = os.Remove(internal.LogFilePath) }()
 	// Create a log file
 	logFile, err := os.OpenFile(internal.LogFilePath, os.O_RDWR|os.O_CREATE, 0666)
 	if err != nil {
 		log.Panic(err)
 	}
-	defer logFile.Close()
+	defer func() { _ = logFile.Close() }()
```

Comment:
Does logging still work after this? If I am reading this right, this would close the log file immediately, and I think that subsequent calls to InfoLog and ErrorLog are depending on it being open. I might be wrong.